### DISCUSSION TITLE
Codespaces - Educational use and billing

### DISCUSSION THREAD

**Discussion by NuttKatyISD** [community/community#66350] [GitHub Education]
### Hi everyone,

I've been using Classroom for the past few years and am looking at adding Codespaces. But I'm a little confused about billing for educational use.

The note on using Codespaces under an educational organization says this.

> The free allowance is estimated to be enough for a class of 50 with 5 assignments per month, on a 2 core machine with 1 codespace stored per student.

What happens when we go over that because I'm pretty sure we will? The only pricing I can find is a per seat, but that's outside of the educational pages.

**Comment by Vijaykv5** [comment]
@NuttKatyISD 
The free allowance you mentioned, which is estimated to cover a class of 50 with 5 assignments per month on specific hardware configurations, is a generous offering from GitHub for educational purposes. This allowance typically includes a certain amount of compute resources and storage space.
If your usage exceeds the free allowance, additional usage may result in charges. GitHub Codespaces pricing is usually based on factors such as the number of seats, compute resources, and storage consumed.

**Reply by NuttKatyISD** [reply]
I agree, it's great they let us use anything free. 

My issue is that if I'm going down this rabbit hole, which I really want to do, I'm going to have to give the people that sign the checks at least a solid estimate of how much it'll cost. Does it roll back to the normal pricing like on https://github.com/pricing/calculator ?

Do we know what the free allotment actually is?
